lib/github-auth.js

```diff
@@ -2,14 +2,15 @@ var querystring = require('querystring');
 var request = require('request');
 var autosave = require('json-autosave');
 var serverSecrets;
+var baseUrl = process.env.BASE_URL || "https://shields.io";
 try {
   // Everything that cannot be checked in but is useful server-side
   // is stored in this JSON data.
   serverSecrets = require('../private/secret.json');
 } catch(e) {}
 var githubUserTokens;
-var githubUserTokensFile = './private/github-user-tokens.json';
-autosave(githubUserTokensFile, {data:[]}).then(function(f) {
+var githubUserTokensFile = './private/.github-user-tokens.json';
```

Comment:
I got the inspiration from this commit, which suggested that it's a hidden file: https://github.com/badges/shields/commit/7c8b0e3d32ee6f33d1568d9278f0cb053224edc8

I'm confused as to what the difference is between a file that is located in `./private/` and one that is located in the root (not in `./public`). Both are not publicly accessible on the web server, right?